c/models/aggregator.cc

```diff
@@ -549,14 +561,17 @@ void Aggregator<T>::group_1d_categorical() {
   auto d_members = static_cast<int32_t*>(dt_members->columns[0]->data_w());
   const int32_t* offsets0 = grpby0.offsets_r();
 
-  #pragma omp parallel for schedule(dynamic)
-  for (size_t i = 0; i < grpby0.ngroups(); ++i) {
-    size_t off_i = static_cast<size_t>(offsets0[i]);
-    size_t off_i1 = static_cast<size_t>(offsets0[i+1]);
-    for (size_t j = off_i; j < off_i1; ++j) {
-      d_members[ri0[j]] = static_cast<int32_t>(i);
-    }
-  }
+  // TODO: use dynamic instead?
```

Comment:
It looks like we don't have many use cases for categorical aggregation, so can leave with static for the moment.